### PR TITLE
DDT-1180 - message routing tables

### PR DESCRIPTION
**Pull request by FinnAtAsseco**

### CODE REVIEW

File: datamodel/initdb.d/03_dcsa_im_v3_0.sql [hunk 1]
```diff
@@ -839,4 +839,24 @@ CREATE TABLE dcsa_im_v3_0.operations_event (
 
 ALTER TABLE dcsa_im_v3_0.operations_event ADD PRIMARY KEY (event_id);
 
+-- DDT-1180 - message routing
```
Comment (HenrikHL) [inline datamodel/initdb.d/03_dcsa_im_v3_0.sql:842]: Please move this to 05 as it is not part of the official IM

Comment (FinnAtAsseco) [reply datamodel/initdb.d/03_dcsa_im_v3_0.sql]: Done

File: datamodel/initdb.d/05_reference_implementation_support.sql [hunk 2]
```diff
@@ -726,4 +726,24 @@ CREATE TABLE dcsa_im_v3_0.vessel_schedule_terminal_visits (
...
 
 
+-- DDT-1180 - message routing
+DROP TABLE IF EXISTS dcsa_im_v3_0.message_routing_rule CASCADE;
+CREATE TABLE dcsa_im_v3_0.message_routing_rule (
+    id uuid NOT NULL PRIMARY KEY DEFAULT uuid_generate_v4(),
+    api_url varchar(255) NOT NULL,
+    login_type varchar(8) NOT NULL CHECK(login_type IN ('OIDC')),
+    login_information jsonb NOT NULL,
+    vessel_mo_number varchar(255) NULL
```
Comment (douradofrazer) [inline datamodel/initdb.d/05_reference_implementation_support.sql:736]: typo, should be `vessel_imo_number`

**Review by douradofrazer — CHANGES_REQUESTED** [review]

File: datamodel/initdb.d/05_reference_implementation_support.sql (hunk 2 above)
Comment (FinnAtAsseco) [reply datamodel/initdb.d/05_reference_implementation_support.sql]: Fixed

**Review by douradofrazer — APPROVED** [review]

**Pull request merged**